src/conductor/client/workflow/executor/workflow_executor.py

```diff
@@ -37,21 +37,30 @@ def start_workflow(self, start_workflow_request: StartWorkflowRequest) -> str:
             body=start_workflow_request,
         )
 
-    def start_workflows(self, quantity: int, workflow_name: str) -> List[str]:
-        """Start `quantity` instances of given workflow name 
+    def start_workflows(self, start_workflow_request: []) -> List[str]:
```

Comment:
What do you think about being a little bit more descriptive for the typing guide? (e.g. `start_workflow_request: List[StartWorkflowRequest]`)